spec/template_string_spec.lua

```diff
@@ -1,6 +1,16 @@
 local TemplateString = require 'apicast.template_string'
+local ngx_variable = require 'apicast.policy.ngx_variable'
+local LinkedList = require('apicast.linked_list')
 
 describe('template string', function()
+  before_each(function()
+    -- Stub the available context to avoid depending on ngx.var.*
+    stub(ngx_variable, 'available_context', function(policies_context)
```

Comment:
Is it necessary to stub this for each test? Can't it be just one that is actually using `a_value` ?

Reply:
Not needed in all the tests, just the ones that use Liquid.
Without this, all the tests that use Liquid will raise an "API disabled" error because `ngx_variable` accesses `ngx.var.*` and `ngx.req.*`.

Reply:
Sure, but each test needs this empty and only one actually needs the `a_value`.

Reply:
I've fixed this.
At first, I thought you were referring to the need for stubbing ngx_variable in general.

Reply:
Regarding the need for stubbing `ngx_variable`, I think it's a good compromise for now. I've tried exploring other solutions like injecting a function that gets the available context for Liquid in the TemplateString modules, but I didn't find any quick solution that will simplify these tests.